dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -151,15 +151,23 @@ var color2name = function (color) {
   return settings.player_color_names[idx];
 };
 
-var Food = function (settings) {
-  if (!(this instanceof Food)) {
-    return new Food();
-  }
-  this.id = settings.id;
-  this.position = settings.position;
-  this.color = settings.color;
-  return this;
-};
+/**
+ * Representation of a game resource, which for the moment is limited to a
+ * simple Food type.
+ *
+ * @param {*} data Information passed from the server, including the type ID
+ * @returns Object with merged server state + object definition values
+ */
+function Resource(data) {
```

Comment:
I guess this should be `Item`? Let's be sure to stay consistent with what's in the python code either way.